src/Pages/Frontpage/Survey/Depricated/Option.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect, useState } from "react";
+import { UserContext } from "../../../../Main/UserContext";
+
+export default (props) => {
+  const [data, setData] = useContext(UserContext).surveyChoices,
+  [checked, setChecked] = useState(false)
+  // console.log(props.text+": "+props.optionIndex)
+  /* 
+  Syncing to context by utilizing context useState hook function, 
+  and checking the context each time. All <Option /> loads every time a change is many in any one componant.
+  TODO: Look into individual <Option /> state (context) managment for less remounting.
+  */
+useEffect(() => {
+  // console.log(data[props.themeIndex])
+  // console.log(props.optionIndex)
+  console.log("------------")
+  console.log(data[props.themeIndex])
+    console.log(props.optionIndex)
+    console.log("-------------")
+  // setChecked(data[props.themeIndex] === props.optionIndex)
+  // console.log(checked)
+},data)
+
+
+  return (
+    <>
```

Comment:
can you do that? :)

Reply:
Called a ghost div :)

Reply:
Ghost div. copies elements into the parent without creating a new div. Lets you cheat the rule of returning only 1 element. Not a good thing to do as a rule. Should be done only in select instances. It does help the designer as there are less layers of divs to worry about.